src/DataLib/fendl32B_retrofit/test_reaction_data.py

```diff
@@ -50,4 +50,44 @@ def test_emission_breakdown(emitted_particles, exp):
 )
 def test_nucleon_changes(emission_dict, exp):
     obs = rxd.nucleon_changes(emission_dict)
-    assert_array_equal(obs, exp)
\ No newline at end of file
+    assert_array_equal(obs, exp)
+
+@pytest.mark.parametrize(
+    "mt_data_csv, exp",
```

Comment:
This is good use of parametrization, but I'm not sure they need to be in different files.  Let's discuss what needs to be tested specifically here that is not tested elsewhere. It needs to test that you can read/process multiple entries. But the processing of each individual entry is already demonstrated in other tests, so it may not be necessary to have that breadth here.  Perhaps a case with a single entry and a case with multiple entries? 

Let me know if you think I'm missing something.

Reply:
> This is good use of parametrization, but I'm not sure they need to be in different files. Let's discuss what needs to be tested specifically here that is not tested elsewhere. It needs to test that you can read/process multiple entries. But the processing of each individual entry is already demonstrated in other tests, so it may not be necessary to have that breadth here. Perhaps a case with a single entry and a case with multiple entries?
> 
> Let me know if you think I'm missing something.

No, that makes sense. I was not sure on the convention for how many tests to include, so that's why I had multiple files, but in principle, they are not accomplishing different things.

Reply:
There is definitely an art testing.  Normally, I don't worry too. much about "too many" tests, but I would like to limit the number of testing files we need